codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/EndpointDisableHttps.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+ package software.amazon.smithy.aws.go.codegen;
+
+ import java.util.ArrayList;
+import java.util.List;
+
+import software.amazon.smithy.codegen.core.Symbol;
+ import software.amazon.smithy.codegen.core.SymbolProvider;
+ import software.amazon.smithy.go.codegen.GoDelegator;
+ import software.amazon.smithy.go.codegen.GoSettings;
+ import software.amazon.smithy.go.codegen.GoStackStepMiddlewareGenerator;
+ import software.amazon.smithy.go.codegen.GoWriter;
+ import software.amazon.smithy.go.codegen.MiddlewareIdentifier;
+ import software.amazon.smithy.go.codegen.SmithyGoDependency;
+ import software.amazon.smithy.go.codegen.SymbolUtils;
+ import software.amazon.smithy.go.codegen.endpoints.EndpointMiddlewareGenerator;
+ import software.amazon.smithy.go.codegen.integration.GoIntegration;
+import software.amazon.smithy.go.codegen.integration.MiddlewareRegistrar;
+import software.amazon.smithy.go.codegen.integration.RuntimeClientPlugin;
+import software.amazon.smithy.model.Model;
+ import software.amazon.smithy.model.knowledge.TopDownIndex;
+ import software.amazon.smithy.model.shapes.OperationShape;
+ import software.amazon.smithy.model.shapes.ToShapeId;
+import software.amazon.smithy.utils.ListUtils;
```

Comment:
Weird spacing throughout the whole file.

Reply:
fixed